### DISCUSSION TITLE
CLI to update configuration

### DISCUSSION THREAD

**Discussion by alienman5k** [ghostty-org/ghostty#3644] [Feature Requests, Ideas] [answered]
Allow updating configuration for the current running terminal with CLI. One great feature of other terminals is to allow updating the current configuration without the need to change the current config file, this can allow to test things quickly but also have scripts for when you need a specific configuration (like sharing screen or presenting).

**Comment by mitchellh** [comment, marked as answer]
Dupe #2353 (a necessary requirement for this and an obvious thing to do with it)
